src/rendition-mixin.js

```diff
@@ -11,21 +11,22 @@
  * or if undefined returns the current enabled-state for the playlist
  * @return {Boolean} The current enabled-state of the playlist
  */
-let enableFunction = (playlist, changePlaylistFn, enable) => {
-  let currentlyEnabled = typeof playlist.excludeUntil === 'undefined' ||
-                         playlist.excludeUntil <= Date.now();
+let enableFunction = (loader, playlistUri, changePlaylistFn, enable) => {
+  let playlist = loader.master.playlists[playlistUri];
+  let blacklisted = playlist.excludeUntil && playlist.excludeUntil > Date.now();
+  let currentlyEnabled = (!playlist.disabled && !blacklisted);
```

Comment:
The function is not as useful here since we still need to get the intermediate information of whether the playlist is blacklisted or not. In the other places we only care about if the playlist is enabled or not, but here we want to make sure that the playlist isn't blacklisted even if `enable` differs from `currentlyEnabled` before calling the `changePlaylistFn`